packages/adena-module/src/transaction/message-info/message-info-vm.ts

```diff
@@ -80,3 +86,16 @@ export const encodeVmAddPackage = (writer: protobuf.Writer, messageInfo: VmAddPa
   }
   return writer;
 };
+
+export const encodeVmRun = (writer: protobuf.Writer, messageInfo: VmRun) => {
+  if (messageInfo.caller !== '') {
+    writer.uint32(10).string(messageInfo.caller);
+  }
+  if (messageInfo.send !== '') {
+    writer.uint32(18).string(messageInfo.send);
+  }
+  if (messageInfo.package !== undefined) {
+    encodePackage(writer.uint32(18).fork(), messageInfo.package).ldelim();
```

Comment:
This simply seems to be the problem.
Change the size of the buffer in the 3rd argument.
`18` -> `26`

Reply:
@luciorubeens seems like protobuf has certain buf size value for nth args.
https://www.tabnine.com/code/javascript/functions/protobufjs/Writer/string

Reply:
Perfect, thank you guys, now it works

<img width="538" alt="CleanShot 2023-10-20 at 10 00 54@2x" src="https://github.com/onbloc/adena-wallet/assets/4539235/90a39cd5-e106-48b9-9da0-e1928c11824b">